spec/scene-perception.html

```diff
@@ -1194,6 +1194,146 @@
+          var initConfig = {
+            useOpenCVCoordinateSystem: true,
+            colorCaptureSize: {'width':320, 'height':240},
+            depthCaptureSize: {'width':320, 'height':240},
+            captureFramerate: 60
+          };
+          sp.init(initConfig).then(function() {
+            // Set the state of buttons.
+            resetButtonState(false);
+
+            // Other initialization work can be done here.
+            console.log('init succeeds');
+          }, function(e) {console.log(e);});
+        };
+
+        startButton.onclick = function(e) {
+          sp.start().then(function() {
+            startButton.disabled = true;
+            stopButton.disabled = false;
+            console.log('SP started successfully');
+          }, function(e) {console.log(e);});
+        };
+
+        stopButton.onclick = function(e) {
+          sp.stop().then(function() {
+            console.log('SP stops working.');
+            startButton.disabled = false;
+            stopButton.disabled = true;
+          }, function(e) {console.log(e);});
+        };
+
+        destroyButton.onclick = function(e) {
+          sp.destroy().then(function() {
+            console.log('stop succeeds');
+            resetButtonState(true);
+          }, function(e) {console.log(e);});
+        };
+      </pre>
+      <h3>
+        Get data from SP module by binding event handlers.
+      </h3>
+      <pre class='example highlight'>
+        var qualityElement = document.getElementById('quality');
+        var accuracyElement = document.getElementById('accuracy');
+
+        sp.onchecking = function(e) {
+          var quality = e.data.quality;
+          qualityElement.innerHTML = 'Quality: ' + quality.toFixed(2);
+        };
+
+        // Please refer the SampleProcessedEvent inferface for event data.
+        sp.onsampleprocessed = function(e) {
+          accuracyElement.innerHTML = 'Accuracy: ' + e.data.accuracy;
+          qualityElement.innerHTML = 'Quality: ' + e.data.quality.toFixed(2);
+          sp.getSample().then(function(sample) {
+            // Show the color and depth image, please refer getSample interface for more details.
+            showColorImage(sample.color);
+            showDepthImage(sample.depth);
+          }, function(e) {console.log(e);});
+          sp.queryVolumePreview(e.data.cameraPose).then(function(volumePreview) {
+            // Show the volume preview image.
+            // Please refer VolumePreviewData interface for detail data format.
+            showVolumePreviewImage(volumePreview);
+          });
+        };
+
+        sp.onmeshupdated = function(e) {
+          thisObj = this;
+          sp.getMeshData().then(function(meshes) {
+            // In the meshes, there are information about vertices, faces, color.
+            // Please refer MeshData inferface for more details.
+            var func = reconstructScene.bind(thisObj, meshes);
```

Comment:
The `reconstructScene` function without code might confuse audience. I think comments are good enough.